.github/workflows/gh-linux_mac.yml

```diff
@@ -0,0 +1,360 @@
+name: builds
+on:
+  push:
+#    branches:
+#      - dev
+#      - release-*
+#      - refs/tags/*
+  pull_request:
+    branches:
+      - '*'  # must quote since "*" is a YAML reserved character; we want a string
+env:
+  BACKWARDS_COMPATIBILITY_ARRAYS: OFF
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    if: ${{ startsWith(github.ref , 'refs/tags') != true && startsWith(github.ref , 'build-') != true }}
+    timeout-minutes: 90
+    strategy:
```

Comment:
Add a `fail-fast: false` so behavior is similar to Azure Pipelines. One failure shouldn't fail all of them.